tests/storage/cbt/pull_collect_runner.py

```
@@ -0,0 +1,190 @@
+        )
+
+    existing_raw_files = _list_client_backup_raw_files()
+    if not params["force_full_backup"] and existing_raw_files:
+        # List is ascending by checkpoint timestamp; seed from the newest prior raw so
+        # incremental extents (since the previous checkpoint) overlay the right base.
+        base_raw_file = existing_raw_files[-1]
+        print(
+            f"Pull incremental collect: seeding {raw_file} from {base_raw_file}",
+            flush=True,
+        )
+        subprocess.run(["cp", base_raw_file, raw_file], check=True)
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)